src/test/scala/cognite/spark/v1/AssetsRelationTest.scala

```diff
@@ -73,6 +74,41 @@ class AssetsRelationTest extends FlatSpec with Matchers with ParallelTestExecuti
     assert(assetsRead == 1)
   }
 
+  it should "support pushdown filters on labels" taggedAs WriteTest in {
```

Comment:
@hakontro  This test passes if I only run AssetsRelationsTest but if I run all tests, it fails at line 91. Would you have a guess regarding the reason? 

I couldn't spot any part affecting this in the other tests, I thought maybe it was because of parallel testing. But I am sure that this sample is created and can be queried as follows according to my experiments in ammonite.

Reply:
It worked this time, is it about default query limit? I saw a similar unpredictable behaviour in one of the timeseries tests (My changes does not affect it actually). I will try to reproduce it.

Reply:
Error I sometimes get in TimeseriesRelationsTest:

[info] - should support upsert in savemode *** FAILED ***
[info]   429 did not equal 404 (TimeSeriesRelationTest.scala:846)

Reply:
This is related to eventual consistency.. sometimes the result of a change to data in CDF won't be present in what you read back if you read it back immediately.. i.e if can create an asset and then immediately try to read it you'll sometimes get a result without that asset. This is because it takes time for servers in the CDF backend to synchronize. For this reason we do retries in our tests, check out the `retryWhile` function if you haven't already.

Anyways, this sometimes fails some of our tests anyways, which is what you're seeing here.